src/classes/Recipe.js

```diff
@@ -1,74 +1,38 @@
+import ingredientsData from '../data/ingredients';
 
-const RecipeBox = require('./RecipeBox');
-const Ingredient = require('./Ingredient');
-const { ids } = require('webpack');
 
 class Recipe {
-  constructor(id, image, ingredients, instructions, name, tags) {
-    this.id = id;
-    this.image = image;
-    this.ingredients = ingredients;
-    this.instructions = instructions;
-    this.name = name;
-    this.tags = tags;
+  constructor(recipeData) {
+    this.id = recipeData.id;
+    this.image = recipeData.image;
+    this.ingredients = recipeData.ingredients;
+    this.instructions = recipeData.instructions;
+    this.name = recipeData.name;
+    this.tags = recipeData.tags;
   }
 
-  determineIngredients(recipeData) {
-   let ingredientIds = recipeData.map(recipe => recipe.ingredients.filter(ingredient => ingredient.id))
-   return ingredientIds;
+  findRecipeIngredientInfo() {
+    const recipeIngredients = this.ingredients.map(recipeIngredient => {
+      const ingredientNames = ingredientsData.find(ingredientItem => ingredientItem.id === recipeIngredient.id);
+      return ingredientNames.name;
+    });
+    return recipeIngredients;
   }
-  determineIngredientsEvenMore() {
-  const ingredientsList = new IngredientsRepo();
-  let ingredientIds = this.determineIngredients()
 
-  const ingredientName = ingredientsList.ingredients.filter(ingredient => {
-    if(ingredientIds.includes(ingredient.id)) {
-      return ingredient.name;
-    }
-    return ingredientName;
-  })
- 
+  getRecipeInstructions() {
+    return this.instructions;
   }
-
-  //split this method
-
-  // input: the ids for the specific ingredient of a recipe
-  // output: the name of the specific ingredient based on the incoming id!
-
-  // take in a parameter that SHOULD be an array that has the ingredient ids
-  // instantiate a class of ingredients, 
-  // if the ingredients.id from instance INCLUDES any of the ids from the 
-  // parameter, return the NAME
-
-  //Returns ingredients based on recipe id
-  determineCost(recipeId) {
-    const ingredient = new Ingredient();
-    let result = this.ingredients.map(ingredient => ingredient.amount)
-    .map((amount) => amount * ingredient.estimatedCostInCents)
+  
+  calculateRecipeCost() {
```

Comment:
I would love to talk through this function.

Reply:
Absolutely! I'd be happy to walk through it with you when we meet up if that works for you.